utils.py

```diff
@@ -189,16 +189,63 @@ def validate_cookies_del(flows: list[dict]) -> None:
             )
 
 
-def flows_to_log(logger_fun: Callable, message: str, flow_dict: dict[str, list]):
-    """Log flows, maximun flows in a log is 200"""
-    length_msg = f"total_length: {len(flow_dict['flows'])}, "
+def _flows_to_log(
+    logger_fun: Callable,
+    flows: list,
+    dpid: Union[str, list],
+    counter: int,
+    flows_acc: dict,
+    maximun: int,
+    message: str,
+    total_length: str,
+) -> tuple[int, dict, str]:
+    start, end = 0, counter
+    while flows[start:end]:
+        counter -= len(flows[start:end])
+        flows_acc[str(dpid)] = flows[start: end]
+        dpids = list(flows_acc.keys()) if isinstance(dpid, str) else dpid
+        if counter == 0:
+            counter = maximun
+            logger_fun(f"{message}{dpids}{total_length}, "
+                       f" flows({counter}): {flows_acc}")
+            total_length = ""
+            flows_acc = {}
+        start += len(flows[start:end])
+        end += counter
+    return counter, flows_acc, total_length
+
+
+def flows_to_log(
```

Comment:
flow_to_log can you rename `dpid` as `dpids` since it's a list to make it immediately clear, the type hint already makes it clear, but even for more clarity, since there are other dpid vars around.

Reply:
Method no longer necessary, removed.